packages/members-api/index.js

```diff
@@ -140,6 +124,29 @@ module.exports = function MembersApi({
         return encodeIdentityToken({sub: member.email});
     }
 
+    async function setMemberGeolocationFromIp(email, ip) {
```

Comment:
Does it make sense to have this method here? Would it make more sense in the `membersApi.members` object?

Reply:
I think either works - I would err toward having it on `membersApi.members`, I _think_ - BUT - this file got way bigger than it was supposed to tbh, and deffo needs splitting out (not as part of this PR).

I don't think it's a big deal to leave this as is :+1: